### DISCUSSION TITLE
Switching from axios to unjs/ofetch

### DISCUSSION THREAD

**Discussion by dantio** [hendt/ebay-api#151] [Ideas]
I'm planing to switch from axios to unjs/ofetch. The fetch lib is better supported in CloudFlare workers and edge functions. This will also make the build smaller.
Please tell me how you use axios (maybe the interceptors?) to make the ebay-api backwards compatible.
And, do you have any objections or other suggestions?

@olliechick 
@iamsr 
@juanbits 
@bennyk8y 
@spidgorny 
@K3vChan 
@raymondma1
@Borduhh
@nrodriguez 

Thank you for all your PRs!

**Comment by K3vChan** [comment]
My project does not use axios intereceptors or do I use this library in any special way. Changing the lib behind the scenes probably wouldn't do me any harm in the next major release.
